### PR TITLE
Explicitly create proxies with 2048 bits

### LINKED ISSUE #11005: WMCore/Credential creates 1024-bit proxies
**Impact of the bug**
services using WMCore/Credential create 1024 bits proxies, which are not good anymore.
There is no celear evidence of problems since other services (HTCondor e.g.) create new proxies
with proper length, but lately some FTS xfers are demonstrated to have 1024-bit proxy
which result in failed transers, see this thread
https://cms-talk.web.cern.ch/t/repost-fts-proxy-issues-it-uses-1024-bits/6827
and even if the full proxy chain in those FTS transfers is not understood, having a fully 2048 bit
chain on our side can only help, if nothing else, to avoid confusion

**Describe the bug**
current code forces a 1024 bit proxy when applying VOMS extensions  to credential retrieved from myproxy
https://github.com/dmwm/WMCore/blob/fbca8729b8aaabb646e87139453dfac70eae7f70/src/python/WMCore/Credential/Proxy.py#L701

**How to reproduce it**
no need

**Expected behavior**
proxies should be 2048 bit to be accepted by CentOS8 servers

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #11005 
Complement to https://github.com/dmwm/WMCore/pull/11006

#### Status
ready

#### Description
Ensure that proxies created in the WMAgent nodes are made with 2048 bits in key.

#### Is it backward compatible (if not, which system it affects?)
YES

#### Related PRs
https://github.com/dmwm/WMCore/pull/11006

#### External dependencies / deployment changes
None

### CODE REVIEW

**Comment by amaltaro** [comment]
Tested in submit8 at FNAL

**Review by todor-ivanov — APPROVED** [review]
looks  good

Thanks @amaltaro

**Comment by vkuznet** [comment]
@amaltaro what the point of making request review if you merge issue without waiting for review?

**Comment by amaltaro** [comment]
@vkuznet this is exactly what I have been trying to discuss with you and Todor. Depending on the PR, I would say that one review is good enough and there is no need to wait for a second review. It also speeds up the review process, in case one of the reviewers is busy with other activities.

**Pull request merged**
